app/cafe.py

```diff
@@ -0,0 +1,23 @@
+import datetime
+from app.errors import (NotVaccinatedError, OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("Visitor is not vaccinated")
+
+        vaccine_info = visitor["vaccine"]
+        expiration_date = vaccine_info.get("expiration_date")
+
+        if expiration_date < datetime.date.today():
```

Comment:
You are comparing the expiration date of the vaccine with the current date. However, you didn't check if the expiration date is `None`. If the `expiration_date` field is missing in the `vaccine_info` dictionary, the `get` method will return `None`, and comparing `None` with a date will raise a `TypeError`. You should add a check for `None` before this line.